packages/inscription-protocol/src/defaults/common-defaults.ts

```diff
@@ -132,7 +132,7 @@ export const DEFAULT_WEB_SERVICE_DATA: WebserviceStartData = {
   permission: {
     anonymous: true,
     error: IVY_EXCEPTIONS.security,
-    role: 'Everyone'
+    roles: ['Everyone']
```

Comment:
I'm confused. Why do we have here `Everyone` ? @ivy-rew

Reply:
Just to be clear, not sure if this is the thing you are confused about, this common-defauls are only used for unit tests and the mock integration.
If the problem is why the webservice start has the role everybody and the real default is something else, it is maybe just for easier testing that both starts behave the same.

Reply:
ahh these are only used for tests! ok I'm fine :)